platform/linux-generic/include/odp_classification_datamodel.h

```diff
@@ -24,7 +24,7 @@ extern "C" {
 #include <odp_pool_internal.h>
 #include <odp_packet_internal.h>
 #include <odp_packet_io_internal.h>
-#include <odp_queue_internal.h>
+#include <odp_schedule_if.h>
```

Comment:
I added queue things to odp_schedule_if.h because queues and scheduler are tightly coupled at the implementation level. It does not make sense to rename odp_schedule_if.h to odp_component_if.h as that file will become too big. I had similar concerns from Brian as well.

Instead, I will try to move queue related things from odp_schedule_if.h/.c to odp_queue_if.h/odp_queue_if.c (new files)